File: app/src/main/java/com/example/task1/Student.java
package com.example.task1;

public class Student {
    private String name, birthDay, phoneNumber, field, level;

    public Student(String name, String birthDay, String phoneNumber, String field, String level) {
        this.name = name;
        this.birthDay = birthDay;
        this.phoneNumber = phoneNumber;
        this.field = field;
        this.level = level;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getField() {
        return field;
    }

    public void setField(String area) {
        this.field = field;
    }

    public String getBirthDay() {
        return birthDay;
    }

    public void setBirthDay(String birthDay) {
        this.birthDay = birthDay;
    }

    public String getLevel() {
        return level;
    }

    public void setLevel(String level) {
        this.level = level;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

}

File: app/src/main/java/com/example/taskjavacore/Student.kt
package com.example.taskjavacore

class Student(
    var name: String, var yearOfBirth: String,
    var phoneNumber: String, var field: String,
    var level: String
) {
    fun checkFind(string: String): Boolean {
        if (name.contains(string, true) || yearOfBirth.contains(string, true)
            || phoneNumber.contains(string, true) || field.contains(string, true)
            || level.contains(string, true)
        ) {
            return true
        }
        return false
    }
}